tests/unit/hive_metastore/test_catalog_schema.py

```diff
@@ -100,32 +100,33 @@ def raise_catalog_exists(catalog: str, *_, **__) -> None:
     )
     table_mapping = TableMapping(installation, ws, backend)
     principal_acl = create_autospec(PrincipalACL)
-    hive_acl = create_autospec(GrantsCrawler)
-    grants = [
-        Grant('user1', 'SELECT', 'catalog1', 'schema3', 'table'),
-        Grant('user1', 'MODIFY', 'catalog2', 'schema2', 'table'),
-        Grant('user1', 'SELECT', 'catalog2', 'schema3', 'table2'),
-        Grant('user1', 'USAGE', 'hive_metastore', 'schema3'),
-        Grant('user1', 'DENY', 'hive_metastore', 'schema2'),
+    interactive_cluster_grants = [
+        Grant('princ1', 'SELECT', 'catalog1', 'schema3', 'table'),
```

Comment:
The names were confusing, because the interactive cluster grants are for principals not users. And the hive grants can be for users, principals and groups

Reply:
you're confused ;) principal is an alias for group or user or service principal.